modules/oadp-debugging-oc-cli.adoc

```diff
@@ -34,33 +34,33 @@ $ oc logs pod/<velero>
 [id="oc-velero-debug-logs_{context}"]
 == Velero pod debug logs
 
-Use the `oc edit` command to set the `Velero` pod logs to debug level:
+You can specify the Velero log level in the DataProtectionApplication resource as shown in the example below.
 
-. Edit the `Velero` deployment:
-+
-[source,terminalsubs="attributes+"]
-----
-$ oc edit deployment/velero -n {namespace}
-----
+[NOTE]
+====
+This option is only available in OADP version 1.0.3.
```

Comment:
better said: **from** 1.0.3 and not **only available in**...

Reply:
@mperetzred
But will it be available in the next version?

Reply:
yea, it's a new feature. It's suppose to be available from now on...